tests/test_context_processors.py

```diff
@@ -19,3 +19,15 @@ def test_login_redirect_unicode_quote(self):
                 "REDIRECT_QUERYSTRING": "next=profile/sj%C3%B3",
             },
         )
+
+    def test_login_redirect_malformed_post(self):
+        request = self.request_factory.post("/", data="no boundary", content_type="multipart/form-data")
```

Comment:
Needs formatting, see https://results.pre-commit.ci/run/github/69119240/1696338661.I5RZqwrgRPmTEG5Or7dWkw (or enable editing of the PR and pre-commit.ci will do the fixes for you).

Reply:
Sorry, forgot to setup pre-commit locally.